wavefront/server/modules/voice_agents_module/voice_agents_module/models/tool_schemas.py

```
@@ -0,0 +1,294 @@
+    parameter_schema: Union[Dict[str, Any], None, Any] = Field(default=UNSET)
+    response_template: Union[str, None, Any] = Field(default=UNSET)
+
+    @field_validator('name')
+    @classmethod
+    def validate_name(cls, v):
+        if v is UNSET:
+            return v
+        if not re.match(r'^[a-z_][a-z0-9_]*$', v):
+            raise ValueError(
+                'Name must follow Python function naming: lowercase letters, numbers, underscores only'
+            )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)